src/WarcraftLegacies.Source/Quests/BlackEmpire/QuestMawofGorma.cs

```diff
@@ -0,0 +1,55 @@
+﻿using System.Collections.Generic;
+using MacroTools.Extensions;
+using MacroTools.FactionSystem;
+using MacroTools.ObjectiveSystem.Objectives.ControlPointBased;
+using MacroTools.ObjectiveSystem.Objectives.FactionBased;
+using MacroTools.ObjectiveSystem.Objectives.TimeBased;
+using MacroTools.QuestSystem;
+using WCSharp.Shared.Data;
+
+namespace WarcraftLegacies.Source.Quests.BlackEmpire
+{ 
+  /// <summary>
+  /// Kill some creeps to gain an outpost and 1 forgotten one.
+  /// </summary>
+  public sealed class QuestMawofGorma : QuestData
+  {
+    private readonly List<unit> _rescueUnits;
+
+    /// <summary>
+    /// Initializes a new instance of the <see cref="QuestMawofGorma"/> class.
+    /// </summary>
+    /// <param name="rescueRect">Units in this area will start invulnerable and be rescued when the quest is complete.</param>
+    public QuestMawofGorma(Rectangle rescueRect) : base("The Maw of Gor'ma",
```

Comment:
```suggestion
    public QuestMawofGorma(Rectangle rescueRect) : base("Maw of Gor'ma",
```
(Just because the other Maw quests don't have "the")